internal/rbd/rbd_util.go

```diff
@@ -1882,3 +1893,40 @@ func genVolFromVolIDWithMigration(
 
 	return rv, err
 }
+
+// Set PV/PVC/PVCNamespace metadata on RBD image.
+func (rv *rbdVolume) setImageMetadata(
```

Comment:
you might need to rename the funtion as this is specific to setting PVC-related information?